WhatsApp/app/src/main/res/layout/fragment_settings_chat.xml

```diff
@@ -267,8 +270,9 @@
                 android:layout_height="wrap_content"
                 android:layout_below="@+id/tvArchivedChats"
                 android:layout_alignParentStart="true"
-                android:layout_toEndOf="@id/archivedSwitch"
                 android:layout_marginStart="70dp"
+                android:layout_marginEnd="140dp"
+                android:layout_toEndOf="@id/archivedSwitch"
                 android:text="@string/archivedChatsDesc" />
         </RelativeLayout>
```

Comment:
for this relative layout, here is an example of how to use the alignment properties. Please do the same for every row and keep the marginEnd to max 20dp.

`        <RelativeLayout
            android:id="@+id/archivedChatsLayout"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:clickable="true"
            android:focusable="true"
            android:paddingTop="12dp"
            android:paddingBottom="12dp"
            android:background="?attr/selectableItemBackground">

            <com.google.android.material.switchmaterial.SwitchMaterial
                android:id="@+id/archivedSwitch"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_alignParentRight="true"
                android:checked="false" />


            <TextView
                android:id="@+id/tvArchivedChats"
                style="@style/chatSettingsText"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_alignParentStart="true"
                android:layout_marginStart="70dp"
                android:layout_toStartOf="@id/archivedSwitch"
                android:text="@string/archivedChatsTitle" />

            <TextView
                android:id="@+id/tvArchivedChatsDesc"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_below="@+id/tvArchivedChats"
                android:layout_alignParentStart="true"
                android:layout_marginStart="70dp"
                android:layout_marginEnd="20dp"
                android:layout_toStartOf="@id/archivedSwitch"
                android:text="@string/archivedChatsDesc" />
        </RelativeLayout>`